django_components/component.py

```diff
@@ -67,20 +67,20 @@ class Component(View, metaclass=SimplifiedInterfaceMediaDefiningClass):
     # Either template_name or template must be set on subclass OR subclass must implement get_template() with
     # non-null return.
     template_name: ClassVar[Optional[str]] = None
-    template: ClassVar[Optional[str]] = None
-    js: ClassVar[Optional[str]] = None
-    css: ClassVar[Optional[str]] = None
+    template: Optional[str] = None
+    js: Optional[str] = None
+    css: Optional[str] = None
```

Comment:
Removed `ClassVar` annotation from component media & template.

This was causing MyPy errors when using the [inlined / single-file component method](https://github.com/EmilStenstrom/django-components#using-single-file-components) like in [this example](https://github.com/JuroOravec/django-components/blob/5c5d80b57dea6932fb3cb134e28829dd96159761/sampleproject/components/greeting.py#L32)